src/auro-datetime.js

```diff
@@ -154,10 +156,24 @@ class AuroDatetime extends LitElement {
       toLowerCase();
   }
 
+  /**
+   * @private Internal function to generate proper time zone local.
+   * @returns {string} - Date/Time zone string.
+   * @param {string} template - Determines which template model to use.
+   */
+  tzTime(template) {
+
+    const scrubTimeZone = this.setDate.slice(0, -6);
+    const newDateTime = new Date(scrubTimeZone);
+
+    return newDateTime.toLocaleString('en-us', template).replace(/^0+/u, '').
+      toLowerCase();
```

Comment:
This looks a little strange all lowercase (e.g. `thu, jul 14, 2022`), was that intentional?

Reply:
Good catch... that function originally was only being applied to `time`, but then updated to also support date. So... yep, need to fix that.